carma_wm/include/carma_wm/WorldModel.h

```diff
@@ -140,6 +140,13 @@ class WorldModel
    */
   virtual LaneletRouteConstPtr getRoute() const = 0;
 
+    /*! \brief Get trackpos of the end of route point relative to the last lanelet in the route
+   *
+   * \return Trackpos 
+   */
+  virtual TrackPos getRouteEndTrackPos() const = 0;
```

Comment:
I think it would make more sense to use the route track position rather than the lanelet track position. This is because the maneuvers are planned relative to the route track position, so it will take less logic for them to compare against that. 
See carma_wm::geometry::routeTrackPos()

Reply:
using route track position now